charts/emby/1.6.3/README.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
```

Comment:
Don't worry about this file, it gets automatically replaced/removed

Reply:
Should I drop it from the PR?

Reply:
No it needs to be there, don't worry about it was all I was saying.